### PR TITLE
[DocumentIntelligence] Update tsp config for Python

### PR DESCRIPTION
**Pull request by YalinLi0312**
Update Python configuration in tspconfig.yaml as:
* Package version has incremented to `1.0.0b2`.
* Package name should be `Azure AI Document Intelligence`, without it would get `Azure Ai Documentintelligence` which does not look good.

### CODE REVIEW

File: specification/ai/DocumentIntelligence/tspconfig.yaml [hunk 1]
```diff
@@ -18,11 +18,11 @@ options:
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/DocumentIntelligence.json"
   # Uncomment this line and add "@azure-tools/typespec-python" to your package.json to generate Python code
   "@azure-tools/typespec-python":
-    "basic-setup-py": true
-    "package-version": 1.0.0b1
-    "package-dir": "azure-ai-documentintelligence"
-    "package-name": "azure-ai-documentintelligence"
-    # "output-path":
+    package-pprint-name: "\"Azure AI Document Intelligence\""
```
Comment (catalinaperalta) [inline specification/ai/DocumentIntelligence/tspconfig.yaml:21]: Nit the emitter fixed the issue that required the double quotes:
```suggestion
    package-pprint-name: "Azure AI Document Intelligence"
```

File: specification/ai/DocumentIntelligence/tspconfig.yaml [hunk 2]
```diff
@@ -18,11 +18,11 @@ options:
...
+    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
```
Comment (catalinaperalta) [inline specification/ai/DocumentIntelligence/tspconfig.yaml:22]: Also, just checking do you need this property?

Comment (YalinLi0312) [reply specification/ai/DocumentIntelligence/tspconfig.yaml:22]: Let me try to generate without it.

Comment (YalinLi0312) [reply specification/ai/DocumentIntelligence/tspconfig.yaml:22]: No difference, I'll remove this line.

**Review by catalinaperalta — APPROVED** [review]

**Pull request merged**